vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -11101,7 +11101,7 @@ unit:Kilo-FT3
   qudt:hasDimensionVector qkdv:A0E0L3I0M0H0T0D0 ;
   qudt:hasQuantityKind quantitykind:LiquidVolume ;
   qudt:hasQuantityKind quantitykind:Volume ;
-  qudt:symbol "k.ft³" ;
+  qudt:symbol "kft³" ;
```

Comment:
What would be the symbol for (1000 feet)**3, as compared with this unit, which is 1000 (cubic feet)?

Reply:
Either variant of the symbol still leaves me with the question of how to interpret it unambiguously. It seems to me that the only recourse is the use of parentheses, if we are to include a symbol at all. What about k(ft³)?